translator/source_code/combine_wzdx.py

```diff
@@ -0,0 +1,157 @@
+import json
+import copy
+from shapely.geometry.polygon import Polygon
+from shapely.geometry import Point
+import geopy
+from geopy.distance import geodesic
+import pyproj
+import getopt
+import sys
+
+
+def main():
+
+    icone_file, cotrip_file, outputfile = parse_arguments(
+        sys.argv[1:], default_output_file_name='combined_wzdx_message.geojson')
+
+    with open(outputfile, 'w+') as f:
+        wzdx_icone = json.loads(open(
+            icone_file, 'r').read())
+        wzdx_cotrip = json.loads(open(
+            cotrip_file, 'r').read())
+
+        polygon = generate_polygon(
+            wzdx_cotrip['features'][0]['geometry']['coordinates'], 100)
+        feature = iterate_feature(polygon, wzdx_icone)
+        if feature:
+
+            f.write(json.dumps(combine_wzdx(
+                wzdx_cotrip, wzdx_icone, feature), indent=2))
+            print('duplicate iCone and COtrip messages were found and combined WZDx message was written to combined_wzdx_message.geojson file')
```

Comment:
"Duplicate" is not quite the right terminology here, we have overlapping datasets that correspond to the same work zone and provide different data that we are combining into a single new source. Can we update these print messages to be a bit more descriptive with that in mind? Maybe just "Combined iCone and COtrip messages written to xxx".

Reply:
should be fixed